ilc/server.js

```diff
@@ -7,21 +10,40 @@ const serveStatic = require('./serveStatic');
 
 app.get('/ping', (req, res) => res.send('pong'));
 
-
 const tailor = tailorFactory(config.get('registry.address'), config.get('cdnUrl'));
 tailor.on('error', (req, err) => {
     console.error('Tailor error:');
     console.error(err);
 });
 
+app.set('view engine', 'ejs');
+app.set('views', './templates');
 
 app.use('/_ilc/', serveStatic(config.get('productionMode')));
 
+app.get('/_ilc/page/500/:errorId', (req, res) => {
```

Comment:
I think that it's better to pass `errorId` via GET params to make it optional. There may be some cases when we need this page w/o error ID.

Reply:
Updated